src/xpanse-api/generated/models/ServiceVo.ts

```diff
@@ -95,5 +95,8 @@ export namespace ServiceVo {
         DESTROYING = 'DESTROYING',
         DESTROY_SUCCESS = 'DESTROY_SUCCESS',
         DESTROY_FAILED = 'DESTROY_FAILED',
+        MIGRATING = 'MIGRATING',
+        MIGRATE_SUCCESS = 'MIGRATE_SUCCESS',
```

Comment:
these files must not be touched. These are auto generated from the API.
Basically these new values will never come in the API response.